docs/deprecations.md

```diff
@@ -147,17 +147,19 @@ Org, app, service, stage, and region are required to resolve variables when logg
 
 Deprecation code: `LAMBDA_HASHING_VERSION_V2`
 
-Starting with v3.0.0, the default value of `lambdaHashingVersion` will be equal to `20201221`. You can adapt to this behavior now, by setting `provider.lambdaHashingVersion` to `20201221`.
+Resolution of lambda version hashes was improved with better (fixed deterministism issues) algorithm, which will be used starting with v3.0.0
 
-When trying to `sls deploy` for the first time after migration to new `lambdaHashingVersion`, you might encounter an error, similar to the one below:
+You can adapt your services to use it now, by setting `provider.lambdaHashingVersion` to `20201221`.
+
+**Notice:** If you apply this on already deployed service, without any changes to lambda code. You might encounter an error, similar to the one below:
```

Comment:
I think it could be phrased as
"If you apply this on already deployed service without any changes to lambda code, you might encounter an error similar to the one below:" to sound a bit more natural (? in my opinion) - what do you think?

Reply:
@pgrzesik Great suggestion. Updated the doc